### PR TITLE
Upstream/soundwire comments2

### PR DESCRIPTION
**Pull request by plbossart**

### CODE REVIEW

File: drivers/soundwire/cadence_master.c [hunk 1]
```diff
@@ -1280,8 +1280,7 @@ int sdw_cdns_clock_stop(struct sdw_cdns *cdns, bool block_wake)
 	int ret;
 
 	/* Check suspend status */
-	status = cdns_readl(cdns, CDNS_MCP_STAT);
-	if (status & CDNS_MCP_STAT_CLK_STOP) {
+	if (sdw_cdns_is_clock_stop(cdns)) {
```
Comment (lyakh) [inline drivers/soundwire/cadence_master.c:1282]: isn't the status variable now unused?

File: drivers/soundwire/cadence_master.c [hunk 2]
```diff
@@ -1262,15 +1262,7 @@ static const struct sdw_master_port_ops cdns_port_ops = {
...
-	u32 status;
-
-	status = cdns_readl(cdns, CDNS_MCP_STAT) & CDNS_MCP_STAT_CLK_STOP;
-	if (status) {
-		dev_dbg(cdns->dev, "Clock is stopped\n");
-		return true;
-	}
-
-	return false;
+	return !!(cdns_readl(cdns, CDNS_MCP_STAT) & CDNS_MCP_STAT_CLK_STOP);
```
Comment (kv2019i) [inline drivers/soundwire/cadence_master.c:1265]: @plbossart Change is ok, but I'd maybe leave the Titanic reference out from the permanent git log history (maybe leave it as a commit note). But yeah, up to you...

Comment (plbossart) [reply drivers/soundwire/cadence_master.c:1265]: This will be squashed so no issues with rants.

File: drivers/soundwire/cadence_master.c (hunk 1 above)
Comment (plbossart) [reply drivers/soundwire/cadence_master.c:1282]: fixed, thanks

**Pull request merged**